components/ui/tabcounter/src/test/java/mozilla/components/ui/tabcounter/TabCounterTest.kt

```diff
@@ -6,24 +6,28 @@ package mozilla.components.ui.tabcounter
 
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import mozilla.components.support.test.robolectric.testContext
-import mozilla.components.ui.tabcounter.TabCounter.Companion.DEFAULT_TABS_COUNTER_TEXT
 import mozilla.components.ui.tabcounter.TabCounter.Companion.ONE_DIGIT_SIZE_RATIO
 import mozilla.components.ui.tabcounter.TabCounter.Companion.SO_MANY_TABS_OPEN
 import mozilla.components.ui.tabcounter.TabCounter.Companion.TWO_DIGITS_SIZE_RATIO
 import org.junit.Assert.assertEquals
+import org.junit.FixMethodOrder
 import org.junit.Test
 import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
 
+// We need order because we want to run "Default tab count is 0" function first.
 @RunWith(AndroidJUnit4::class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 class TabCounterTest {
 
     @Test
-    fun `Default tab count is a smiley face`() {
+    // This "A" is intended because we want to run this function first
+    fun `A Default tab count is 0`() {
```

Comment:
Each test should be creating its own brand new `tabCounter`.  Does order matter here?  Can you please help investigate this so we don't need the tests to run in a specific order?  Thanks,

Reply:
This didn't worked while i was trying but now they're working, confused :)
Gonna roll it back.